src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
 // write code here
+const ulLists = document.querySelectorAll('li');
```

Comment:
The `querySelectorAll('li')` selects all `<li>` elements in the document. Ensure that this is the intended behavior, as it will apply the functionality to every list item on the page.